packages/manager/apps/dedicated/client/app/billing/autoRenew/agreements/user-agreements.html

```diff
@@ -16,7 +16,18 @@
     </div>
 
     <div data-ng-if="toActivate.count || (loading && loaders.toActivate)">
-        <h2 data-translate="user_agreements_to_validate"></h2>
+        <div class="row">
+            <h2 class="col" data-translate="user_agreements_to_validate"></h2>
+            <div class="col">
+                <oui-button
+                    class="d-block text-right"
+                    data-ng-if="!loaders.toActivate"
+                    data-on-click="gotoAcceptAllAgreements(toActivate.list.results)"
```

Comment:
acceptAll Agreements are available to all accounts or only to accounts that need migration? Looks like here its for all accounts

Reply:
Since it now shows all the agreements pending (including the normal ones), it is present for all the accounts.